core/controller/src/main/scala/whisk/core/controller/BasicAuthenticationDirective.scala

```diff
@@ -73,7 +74,11 @@ object BasicAuthenticationDirective extends AuthenticationDirectiveProvider {
     }
   }
 
-  def identityByNamespace(namespace: EntityName)(implicit transid: TransactionId, authStore: AuthStore) = {
+  def identityByNamespace(namespace: EntityName)(implicit transid: TransactionId,
+                                                 system: ActorSystem,
+                                                 ec: ExecutionContext,
```

Comment:
`ec` and `logging` seem unnecessary at first glance (the actor system can provide one, and if it's for the datastore, then it should already be configured correctly). Re the logging I thought we've been moving away from this pattern.

Reply:
You are of course right. I have removed the redundant parameters in a second commit.